Lebenslauf.html

```diff
@@ -1,142 +1,152 @@
-    <td>Delegation von Teams mit mehr als fünf Mitarbeitern, Koordination von Veranstaltungsabläufen, Organisation und Nachbereitung vor Ort</td>
-   </tr>
-   </table>
-
-  <h2 style="color:blue;"><h2 style="font-family:verdana;">Ausbildung</h2>
-  <hr>
-
-  <table>
-    <tr>
-      <th>Zeitraum</th>
-      <th>Tätigkeit</th>
-    </tr>
-    <tr>
-      <td>2010-2015</td>
-      <td>B.A. Germanistik und Spanische Philologie, <a href="https://www.uni-potsdam.de/">Universität Potsdam</a></td>
-    </tr>
-    <tr>
-      <td>2009-2010</td>
-      <td>Spanische Philologie und Linguistik, <a href="https://www.uni-potsdam.de/">Universität Potsdam</a></td>
-    </tr>
-    <tr>
-      <td>2008-2009</td>
-      <td>Französische Philologie und Linguistik, <a href="https://www.uni-potsdam.de/">Universität Potsdam</a></td>
-    </tr>
-    <tr>
-      <td>2001-2008</td>
-      <td>Allgemeine Hochschulreife, <a href="https://www.wolkenberg-gymnasium.de/">Wolkenberg Gymnasium Michendorf</a></td>
-     </tr>
-     </table>
-
-  <h3 style="color:green;"><h3 style="font-family:verdana;">Sprachen</h3>
-
-  <table>
-    <tr>
-      <th>Sprache</th>
-      <th>Kenntnisstand</th>
-    </tr>
-    <tr>
-      <td>Deutsch</td>
-      <td>Muttersprache</td>
-    </tr>
-    <tr>
-      <td>Englisch</td>
-      <td>Sehr gute Kenntnisse</td>
-    </tr>
-    <tr>
-      <td>Spanisch</td>
-      <td>Sehr gute Kenntnisse</td>
-    </tr>
-    <tr>
-      <td>Französisch</td>
-      <td>Grundkenntnisse</td>
-    </tr>
-    <tr>
-      <td>Portugiesisch</td>
-      <td>Grundkenntnisse</td>
-    </tr>
-    <tr>
-      <td>Latein</td>
-      <td>kleines Latinum</td>
-     </tr>
-     </table>
-
-  <h3 style="color:green;"><h3 style="font-family:verdana;">Qualifikationen</h3>
-
-  <p>Gitarre (Seit 2005)
-  <br>
-  <p>Chorgesang (Seit 2001)
-
-  <h3 style="color:green;"><h3 style="font-family:verdana;">Kenntnisse</h3>
-
-  <table>
-    <tr>
-      <th>EDV</th>
-      <th>Kenntnisstand</th>
-    </tr>
-   <tr>
-     <td>MS Word 2010</td>
-     <td>Gute Kenntnisse</td>
-   </tr>
-   <tr>
-     <td>MS Excel 2010</td>
-     <td>Grundkenntnisse</td>
-   </tr>
-   <tr>
-     <td>MS Powerpoint 2010</td>
-     <td>Grundkenntnisse</td>
-   </tr>
-   <tr>
-     <td>Markdown</td>
-     <td>Grundkenntnisse</td>
-    </tr>
+
+    <h1 style="font-size:200%; font-family:verdana;">Lebenslauf</h1>
+    <p>Luise Maron</p>
+    <img src="Passfoto.jpg" width="150" height="224">
+    <p>Lennéstraße 19<br> 14469 Potsdam</p>
+    <p>E-Mail: luisemaron@googlemail.com</p>
+    <p>Geburtsort &amp; -datum: Potsdam, den 07. April 1989</p>
+    <p>Staatsangehörigkeit: deutsch</p>
+    <p>Familienstand: ledig</p>
+    <h2 style="color:blue; font-family:verdana;">Berufserfahrung</h2>
+    <hr>
+
+    <table>
+        <tr>
+            <th style="text-align:center;">Zeitraum</th>
+            <th style="text-align:center;">Tätigkeit</th>
+        </tr>
+        <tr>
+            <td><i>2014-heute</i></td>
```

Comment:
There are multiple dashes. The one uses between dates is usually: `&ndash;`. [More here...](https://de.wikipedia.org/wiki/Halbgeviertstrich#Bis-Strich)